com.kaii.hgui.json

```diff
@@ -0,0 +1,65 @@
+{
+    "app-id" : "com.kaii.hgui",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "hgui-rust",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+        ],
+        "env" : {
+            "RUST_BACKTRACE" : "1",
+            "RUST_LOG" : "hgui-rust=debug",
+            "CARGO_HOME" : "/run/build/hgui-rust/cargo"
+        }
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+          "name": "blueprint-compiler",
+          "buildsystem": "meson",
+          "cleanup": ["*"],
+          "sources": [
+            {
+              "type": "git",
+              "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler.git",
+              "tag": "v0.10.0"
+            }
+          ]
+        },
+        {
+            "name" : "hgui-rust",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/kaii-lb/hgui-rust.git",
+                    "commit" : "713af543c6798ee8d96d7ed4d3655453ba7d5ed1"
+                },
+                "generated-sources.json"
```

Comment:
file is missing.

Reply:
i realized, i def shouldn't be doing this so late, but here, this should finally fix it i hope